Media/index.js

```diff
@@ -0,0 +1,160 @@
+// SIDERBAR
+const menuItem = document.querySelectorAll('.menu-item');
+// MESSAGE
+const messagesNotification = document.querySelectorAll('#message-notification');

```

Comment:
querySelectorAll và querySelector khác gì nhau? Trong trường hợp này thì có thể sử dụng getElementById không?

Reply:
querySelecter là sẽ chọn 1 phần tử mà phần tử đó tring khớp trong css
querySlecterAll là sẽ chọn toàn bộ phần tử trùng khớp trong css

còn trong trường hợp này thì ko thể sử dụng getElementById